spec/rakuten_web_service/gora/course_detail_spec.rb

```diff
@@ -0,0 +1,61 @@
+# encoding: utf-8
+
+require 'spec_helper'
+require 'rakuten_web_service'
+
+describe RakutenWebService::Gora::CourseDetail do
+  let(:endpoint) { 'https://app.rakuten.co.jp/services/api/Gora/GoraGolfCourseDetail/20140410' }
+  let(:affiliate_id) { 'dummy_affiliate_id' }
+  let(:application_id) { 'dummy_application_id' }
+  let(:expected_query) do
+    {
+        :affiliateId => affiliate_id,
+        :applicationId => application_id,
+        :golfCourseId => 120092
+    }
+  end
+
+  before do
+    RakutenWebService.configuration do |c|
```

Comment:
Calling `RWS.configuration` with a block will be deprecated from the next release. please use `RWS.configure`.